apps/web/src/app/globals.css

```
@@ -476,6 +476,197 @@
+    width: 5px;
+    height: 5px;
+    --dx: -14px;
+    --dy: -20px;
+  }
+  .flag-celebrate-spark:nth-child(2) {
+    left: 50%;
+    width: 4px;
+    height: 4px;
+    --dx: 2px;
+    --dy: -26px;
+    animation-delay: 60ms;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)